src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -170,7 +173,9 @@ private void getNotifyBoth(boolean self, String query, int offset, int limit,
             notes.setTotalRecords(totalRecords);
             asyncResultHandler.handle(succeededFuture(
               GetNotifyResponse.respond200WithApplicationJson(notes)));
+            logger.info("getNotifyBoth:: returning {} notes of {}", notifylist.size(), totalRecords);
           } else {
+            logger.warn("getNotifyBoth:: exception: " + reply.cause());
```

Comment:
- please don't use concatenation for logging
- please explain what exactly failed when an exception was caught, and please do this whenever you log an exception

Reply:
done